automation_tools/repository.py

```diff
@@ -0,0 +1,156 @@
+"""Taks for managing repositories"""
+from __future__ import print_function
+
+import sys
+
+from automation_tools.utils import distro_info
+from fabric.api import put, run
+
+if sys.version_info[0] == 2:
+    from StringIO import StringIO  # pylint:disable=import-error
+else:
+    from io import StringIO
+
+
+def disable_repos(*args):
+    """Disable repos passed as ``args`` using ``subscription-manager repos
+    --disable``.
+
+    For example::
+
+        disable_repos('repo1', 'repo2')
+
+    Will run the command ``subscription-manager repos --disable repo1 repo2``.
+
+
+    """
+    run('subscription-manager repos --disable {0}'
+        .format(' '.join(['"{0}"'.format(repo) for repo in args])))
+
+
+def delete_custom_repos(**args):
+    """Delete repos files on ``/etc/yum.repos.d``.
+
+    All files that match ``<filename>.repo`` will be deleted, double check when
+    running this task because it can delete repository files created by
+    ``subscription-manager`` and other tools.
+
+    """
+    for name in args:
+        run('rm {0}'.format('/etc/yum.repos.d/{0}.repo'.format(name)),
```

Comment:
Is there any reason that string interpolation is being performed twice here? I imagine it would be more straightforward to write this:

```python
run('rm /etc/yum.repos.d/{0}.repo'.format(name))
```